src/main/java/frc/robot/Constants.java

```diff
@@ -18,6 +18,12 @@
 public final class Constants {
     //controller port
     public static final int controllerport = 0;
+    //port for ball sucker thing in ipor
+    public static final int inputPort = 3;
+    //press button for thing above
+    public static final double  inputSpeed = .6;
```

Comment:
give specific details on the comments. don't use references like "thing" or "above". also the comment above `inputSpeed` does not go with that variable.

Reply:
Agreed. We don't want a book. short is great, but it needs to be concise and detailed.